### ISSUE TITLE
[Feature Request] PvP Rankings

### ISSUE THREAD

**Issue by whitewillem** [RealDeviceMap/RealDeviceMap#37]
**Is your feature request related to a problem? Please describe.**
More and more users are looking for PvP mons. But we dont have any way to show how good mons are or easy filter on them.

**Describe the solution you'd like**
Invent a ranking system based on IV stats for each league and store it in pokemon table / send it in webhook.

**Describe alternatives you've considered**
Implementation in PMSF. But i dont think thats the proper way to do it.

**Comment by 123FLO321** [comment]
How would that be calculated?  @whitewillem

**Comment by whitewillem** [comment]
this is a pretty clean example of how it could be calculated: https://github.com/PartTimeJS/WDR/blob/master/static/PvPRank.js.script

**Comment by jorgitoh** [comment]
I did something similar a little over a year ago. In the first version used calculations but even having very few phones, pokemon and a good memcache the operation was somewhat expensive.

The solution I found, even without being the best, was to set up an auxiliary table only for the great league. This table what I was doing was acting alongside the insert / update of a pokemon taking the corresponding ranking and putting it in a new column automatically. Each pokemon knew where to look for its ranking, that is, if a bulbasaur came out, the corresponding ranking to look for was venusaur, as long as the CP or level is not exceeded.

La tabla auxiliar era enorme. Estoy hablando de 4096 registros por cada Pokemon x 2 si se quiere abarcar Great + Ultra.
